CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# 0.26.0
+
+- Add a message about Bodhi requiring the password and fix construction of the Koji tag for epel branches
```

Comment:
I know these 2 issues were mentioned together in the blog post, but please separate them here. thanks.